bluebottle/payments_flutterwave/models.py

```diff
@@ -0,0 +1,107 @@
+from decimal import Decimal
+import json
+
+from django.db import models
+from django.utils.translation import ugettext as _
+
+from bluebottle.payments.models import Payment
+
+
+class FlutterwavePayment(Payment):
+
+    amount = models.CharField(
+        help_text="Amount",
+        null=True, blank=True,
+        max_length=200)
+    currency = models.CharField(
+        help_text="Transaction currency",
+        default="NGN",
+        null=True, blank=True,
+        max_length=200)
+    auth_model = models.CharField(
+        help_text="Authentication Model - BVN, PIN, NOAUTH, VBVSECURECODE",
+        default="BVN",
+        null=True, blank=True,
+        max_length=200)
+    card_number = models.CharField(
+        help_text="Card Number",
+        null=True, blank=True,
+        max_length=200)
+    cvv = models.CharField(
+        help_text="Card CVV",
+        null=True, blank=True,
+        max_length=200)
+    expiry_month = models.CharField(
+        help_text="Card expiry month",
+        null=True, blank=True,
+        max_length=4)
+    expiry_year = models.CharField(
+        help_text="Card expiry year",
+        null=True, blank=True,
+        max_length=4)
+    bvn = models.CharField(
+        help_text="User BVN, required only for authModel=BVN",
+        default='',
+        null=True, blank=True,
+        max_length=200)
+    pin = models.CharField(
+        help_text="User PIN, required only for authModel=PIN",
+        default='',
+        null=True, blank=True,
+        max_length=200)
+    customer_id = models.CharField(
+        help_text="Customer ID for tracking charge transaction",
+        null=True, blank=True,
+        max_length=100)
+    narration = models.CharField(
+        help_text="Transaction description",
+        null=True, blank=True,
+        max_length=200)
+    response_url = models.CharField(
+        help_text="Callback Url",
+        null=True, blank=True,
+        max_length=200)
+    country = models.CharField(
+        help_text="Country code (NG)",
+        default="NG",
+        null=True, blank=True,
+        max_length=200)
+
+    transaction_reference = models.CharField(
+        help_text="Flutterwave transaction reference",
+        null=True, blank=True,
+        max_length=200)
+
+    response = models.CharField(help_text=_('Response from Flutterwave'),
```

Comment:
can we make this a TextField to avoid truncation of the JSON response if it is greater than 5000 chars ? This also makes sure that we always store the full response from flutterwave and nothing gets lost.